ansible.cfg

```diff
@@ -1,2 +1,2 @@
 [defaults]
-roles_path = ~/.personal_provisioning/roles/basics:~/.personal_provisioning/roles/applications:~/.personal_provisioning/roles/development:~/.personal_provisioning/roles/development_mobile
+roles_path = ~/.personal_provisioning/roles/basics:~/.personal_provisioning/roles/applications:~/.personal_provisioning/roles/development:~/.personal_provisioning/roles/development_mobile:./roles
```

Comment:
Shouldn't be required, because these are just additional role paths. See
https://docs.ansible.com/ansible/intro_configuration.html#roles-path